webapp/components/create_comment.jsx

```diff
@@ -66,11 +69,33 @@ export default class CreateComment extends React.Component {
 
     componentDidMount() {
         PreferenceStore.addChangeListener(this.onPreferenceChange);
+
         this.focusTextbox();
+        document.addEventListener('keydown', this.showShortcuts);
     }
 
     componentWillUnmount() {
         PreferenceStore.removeChangeListener(this.onPreferenceChange);
+        document.removeEventListener('keydown', this.showShortcuts);
+    }
+
+    showShortcuts(e) {
```

Comment:
do we want this here? wouldn't this shortcut trigger the `/shortcuts` command twice? as there already is a listener attach to `document` for the center channel view?